bin/pycbc_make_skymap

```diff
@@ -185,37 +188,48 @@ def main(trig_time, mass1, mass2, spin1z, spin2z, f_low, f_upper, sample_rate,
         command.append("--trigger-time"),command.append(str(np.round(trig_time,5)))
         command.append("--window"),command.append(str(window))
 
-        if custom_frame_files is None or ifo not in custom_frame_files:
-            command.append("--frame-type")
-            command.append(frame_types[ifo])
+        if injection_file:
+            command.append("--injection-file"),command.append(injection_file)
+
+        if fake_strain:
```

Comment:
I just double-checked a couple things and I think this would be more correct like this:
```python
if fake_strain[ifo] is not None:
```